store/articles/list.js

```diff
@@ -14,37 +14,34 @@ const success = storeHelper.createMutationFn(ARTICLE_LIST_RETRIEVED_SUCCESS)
 export const getters = {
   loading: state => state.status === 'loading',
   totalItems: state => state.totalItems,
-  articles: state => state.articles,
+  articles: state => state.articles[state.currentListCode],
   pagesMap: state => state.pagesMap,
   status: state => state.status,
   error: state => state.error,
   limit: state => state.limit,
-  page: state => state.page
+  page: state => state.page,
+  isNext: state => !!(state.pagesMap['hydra:next']) || false,
+  listCode: state => state.currentListCode,
```

Comment:
This should be in some general store I mean currentRoute is definetely something we want to have in a store

Reply:
We talked that we add this change in next PR.